presto-docs/src/main/sphinx/language/time.rst

```diff
@@ -0,0 +1,57 @@
+========================
+Legacy And New Timestamp
+========================
+
+New ``TIMESTAMP`` and ``TIME`` semantics align the types with the SQL standard.
+See the following sections for details.
+
+.. note::
+
+   The new ``TIMESTAMP`` semantics is still experimental. It's recommended to keep
```

Comment:
is -> are

Reply:
https://dictionary.cambridge.org/dictionary/english/semantics suggests `semantics` is uncountable